modules/core/README.md

```diff
@@ -232,11 +232,34 @@ After you have initialised and unsealed Vault, you can use the
 [`nomad-vault-integration`](../nomad-vault-integration) module to Terraform the required policies
 and settings for Vault.
 
+Make sure you have properly configured Vault with the appropriate
+[authentication methods](https://www.vaultproject.io/docs/auth/index.html) so that your users can
+authenticate with Vault to get the necessary tokens and credentials.
+
 The default `user_data` scripts for Nomad servers and clients will automatically detect that the
 policies have been setup and will configure themselves correctly. To update your cluster to use
 the new Vault integration, simply follow the section below to update the Nomad servers first and
 then the clients.
 
+### Nomad ACL
+
+[ACL](https://www.nomadproject.io/guides/acl.html) can be enabled for Nomad so that only users
+with the necessary tokens can submit jobs. This module only enables to built in access controls
```

Comment:
`This module only enables to built in access controls`, is it `This module only enables built-in access controls`?